Dockerfiles/agent-ddot/Dockerfile.agent-otel

```diff
@@ -1,9 +1,11 @@
+ARG AGENT_REPO=datadog/agent-dev
 ARG AGENT_VERSION=nightly-full-main-jmx
```

Comment:
Did you consider using `latest-full` tag which is the latest stable version?

Reply:
We don't want the latest stable version when building on `main`. The latest stable version will be set in the release branches. If you BYOC off main, you will get the latest dev image. Does that make sense?